vendor/k8s.io/client-go/pkg/apis/clientauthentication/OWNERS

```diff
@@ -0,0 +1,9 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
```

Comment:
suspect you need to run ./update-deps.sh (which removes these OWNERS files), but I don't know why prow didn't complain about that already 🤔

Reply:
Hmmm, yeah, will add looking into that to my todo list